src/lib/date.ts

```diff
@@ -203,6 +203,54 @@ function relativeDays(prefix, today, other, max) {
   return null
 }
 
+function relativeDaysUntil(suffix, today, other, max) {
+  const delta = other.diff(today, "days")
+  if (delta >= 0) {
+    if (delta === 0) {
+      return `${suffix} Today`
+    } else if (delta === 1) {
+      return `${suffix} Tomorrow`
+    } else if (delta <= max) {
+      return `${delta} days until ${suffix}`
+    }
+  }
+  return null
+}
+
+function relativeHoursUntil(suffix, today, other) {
```

Comment:
Yea, I'm wondering what you're thinking with this @nspinazz89 . That's why it might be easier to do this all in Force initially (and can move out into Metaphysics when needed to share with Eigen). It can be tricky to know 'up front' exactly what kind of schema you might need for the front-end.

I'm not 100% sure the value of MP returning a string like `"8h 23m Until Lots Begin Closing"` (for instance), cause that seems odd to display as a static string (vs. a countdown timer which has to be a front-end JS component, in which case MP returning this string doesn't seem like it would be helpful).